app/controllers/api/v2/users_controller.rb

```diff
@@ -1,13 +1,38 @@
 module API
   module V2
     class UsersController < API::V2::ApplicationController
-      def show
-        user = User.find(params[:id])
-        authorize user
+      before_action :build_user
+      deserializable_resource :user, only: :update
 
-        render jsonapi: user,
+      def show
+        render jsonapi: @user,
                include: params[:includes]
       end
+
+      def update
```

Comment:
How about we follow the pattern set out here, just on principle?

https://github.com/DFE-Digital/manage-courses-backend/blob/master/app/controllers/api/v2/sites_controller.rb#L16

We don't have validations yet but there could still be errors and, to some extent, it would be nice for the API to be built out a bit more like a platform instead of only feature-by-feature.

Reply:
Done